README.md

```diff
@@ -227,7 +227,7 @@ The git history goes back to 2007 when development moved from svn to hg.
 To see the history before that point, look at
 <https://github.com/sympy/sympy-old>.
 
-You can use git to see the biggest developers. The command:
+You can use git to see the largest contributors. The command:
```

Comment:
As a native English speaker I find both of these descriptions comical.

I'm probably medium-sized (for a man) so I guess that some contributors are smaller than me but probably some are larger. I didn't realise git new which of us was the largest though!

More seriously I'm not sure the purpose of these lines. The best wording is probably "biggest contributors" but I think it would be better not to refer to anything like this at all.

Reply:
@fauzank339 
You should land yourself on a `good first issue` / `easy to solve` issues  by searching in labels . There you can solve legit problems which are meant for new open source contributors . I personally added unit tests to an already solved problem as my first contribution .

Since there is no particular motivation for this pull request ,you can maybe close it !?